ui/components/connections/index.js

```diff
@@ -150,7 +158,77 @@ function Connections({ classes, updateProgress, /*onOpenCreateConnectionModal,*/
   const { notify } = useNotification();
   const StyleClass = useStyles();
 
-  // const statuses = ['ignored', 'connected', 'REGISTERED', 'discovered', 'deleted'];
+  const statuses = ['ignored', 'connected', 'REGISTERED', 'discovered', 'deleted'];
```

Comment:
Use the statuses defined in constants.js

Reply:
switched to enum.js file 👍